src/app/modules/shared/components/details-fields/details-fields.component.ts

```diff
@@ -126,6 +126,7 @@ export class DetailsFieldsComponent implements OnChanges, OnInit {
 
   ngOnChanges(): void {
     this.goingToWorkOnThis = this.entryToEdit ? this.entryToEdit.running : false;
+    this.shouldRestartEntry = false;
```

Comment:
In general it looks good to me... Just wondering (I don't remember) if there is any test for the case when `this.shouldRestartEntry` is `true`. If not, that would be something to implement, please.